src/main/java/com/demoqa/shop/util/RandomString.java

```diff
@@ -0,0 +1,32 @@
+package com.demoqa.shop.util;
+import java.util.Random;
+
+public class RandomString {
+
+    // function to generate a random string of length n
+    public static String getAlphaNumericString(int n) {
+
+        // chose a Character random from this String
```

Comment:
misleading comment as it does not choose anything in this step, only assigns all the alphanumeric characters